setup.sh

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
+
+# This file will be the host environment setup file for all operating systems other than base Windows.
+
+if [[ "$OSTYPE" == "linux-gnu"* ]]; then
+  # Check if root or sudo
+  root=true
+  if [ "$EUID" -ne 0 ]; then
+    root=false
```

Comment:
We exit at the install commands because the pip commands do not need root privileges. Some professional places require IT or admin action to install packages, but not pip. No need to exclude those users.